LDAR_Sim/model_code/ldar_sim.py

```diff
@@ -52,11 +52,12 @@ def __init__(self, state, params, timeseries):
             params['working_directory'] + params['leak_file']).iloc[:, 0])
         state['empirical_sites'] = np.array(pd.read_csv(
             params['working_directory'] + params['vent_file']).iloc[:, 0])
-        if isinstance(params['time_offsite'], str):
-            state['offsite_times'] = np.array(pd.read_csv(
-                params['working_directory'] + params['time_offsite']).iloc[:, 0])
-        else:
-            state['offsite_times'] = np.array([params['time_offsite']])
+        if 'time_offsite' in params:
```

Comment:
This if statement is OK - but really we should kill it and just always have something for this parameter. This is something I will need to clean up, probably no action at this time.

Reply:
created issue #72